src/Symfony/Bundle/SecurityBundle/DependencyInjection/Security/Factory/RememberMeFactory.php

```diff
@@ -304,4 +311,20 @@ private function createTokenProvider(ContainerBuilder $container, string $firewa
 
         return $tokenProviderId;
     }
+
+    private function createTokenVerifier(ContainerBuilder $container, string $firewallName, ?string $serviceId): string
```

Comment:
```suggestion
    private function createTokenVerifier(ContainerBuilder $container, string $firewallName, ?string $serviceId): Reference
```
I'm currently trying your patch, looks good apart from that string cast 👍

Reply:
Woops, thanks for spotting that.